homeassistant/util/color.py

```diff
@@ -307,6 +319,12 @@ def color_RGB_to_hsv(iR: int, iG: int, iB: int) -> Tuple[float, float, float]:
     return round(fHSV[0]*360, 3), round(fHSV[1]*100, 3), round(fHSV[2]*100, 3)
 
 
+# pylint: disable=invalid-sequence-index
+def color_RGB_to_hs(iR: int, iG: int, iB: int) -> Tuple[float, float]:
```

Comment:
I don't mind the conversion in this direction because it does not add in an arbitrary value to make the dimensions match.